### PR TITLE
Do not create a job when action is not create or remove.

### PR DESCRIPTION
**Pull request by ajoaoff**

### CODE REVIEW

**Comment by josemauro** [comment]
Is this PR related to https://github.com/kytos/kytos/issues/1027 ?

**Comment by ajoaoff** [comment]
@josemauro no, it is not.

**Comment by josemauro** [comment]
Okay, thanks. I accidentally delete my comment. (https://github.com/kytos/kytos/issues/1027)

File: scheduler.py [hunk 1]
```diff
@@ -91,11 +91,12 @@ def add_circuit_job(self, circuit, circuit_scheduler):
         :param circuit_scheduler (CircuitSchedule): Circuit schedule data
         :return:
         """
-        job_call = None
         if circuit_scheduler.action == 'create':
             job_call = circuit.deploy
         elif circuit_scheduler.action == 'remove':
             job_call = circuit.remove
+        else:
+            return
```
Comment (hdiogenes) [inline scheduler.py]: Should this really go silent, or should we raise an exception here?

File: scheduler.py [hunk 2]
```diff
@@ -91,11 +91,14 @@ def add_circuit_job(self, circuit, circuit_scheduler):
...
         """
-        job_call = None
         if circuit_scheduler.action == 'create':
             job_call = circuit.deploy
         elif circuit_scheduler.action == 'remove':
             job_call = circuit.remove
+        else:
+            valid_actions = ("'create'", "'remove'")
+            raise ValueError(f'Scheduler action must be '
+                             f'{" or ".join(valid_actions)}')
```
Comment (hdiogenes) [inline scheduler.py:101]:
```suggestion
            raise ValueError("Scheduler action must be 'create' or 'remove'")
```

**Pull request merged**